packages/react-components/src/components/xy-output-component.tsx

```diff
@@ -232,6 +232,16 @@ export class XYOutputComponent extends AbstractTreeOutputComponent<AbstractOutpu
                 }
             </React.Fragment>;
         }
+        if (this.state.outputStatus === ResponseStatus.COMPLETED && this.state.xyData?.datasets?.length===0 ) {
+            return <React.Fragment>
+                {
+                    <p className='no-selected' style={{fontSize: 20, marginRight: '5px', marginLeft: '5px', justifyContent:'center', alignItems:'center'}}>
```

Comment:
I see the following error in the console:
```
root ERROR Warning: validateDOMNesting(...): <p> cannot appear as a descendant of <p>.
    in p (created by XYOutputComponent)
    in p (created by XYOutputComponent)
    in div (created by XYOutputComponent)
    in div (created by XYOutputComponent)
    in div (created by XYOutputComponent)
    in XYOutputComponent (created by TraceContextComponent)
    in Resizable (created by GridItem)
    in DraggableCore (created by GridItem)
    in GridItem (created by ReactGridLayout)
    in div (created by ReactGridLayout)
    in ReactGridLayout (created by ResponsiveReactGridLayout)
    in ResponsiveReactGridLayout (created by WidthProvider)
    in WidthProvider (created by TraceContextComponent)
    in div (created by TraceContextComponent)
    in TraceContextComponent
    in div
```

Reply:
Huh... I'll remove it then... it works though. ?!?